lib/topological_inventory/persister/logging.rb

```diff
@@ -7,7 +7,7 @@ class << self
     end
 
     def self.logger
-      @logger ||= ManageIQ::Loggers::CloudWatch.new
+      @logger ||= ManageIQ::Loggers::CloudWatch.new.tap { |log| log.level = ENV["LOG_LEVEL"] || "DEBUG" }
```

Comment:
@lindgrenj6 not sure if DEBUG should be default, now it seems that INFO is default in kibana

Reply:
Updated to INFO.